plugin/trino-resource-group-managers/src/main/resources/db/migration/postgresql/V1__add_resource_groups_schema.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE IF NOT EXISTS resource_groups_global_properties (
+    name VARCHAR(128) NOT NULL PRIMARY KEY,
+    value VARCHAR(512) NULL,
+    CHECK (name in ('cpu_quota_period'))
+);
+
+CREATE TABLE IF NOT EXISTS resource_groups (
+    resource_group_id BIGSERIAL PRIMARY KEY,
+    name VARCHAR(250) NOT NULL,
+    soft_memory_limit VARCHAR(128) NOT NULL,
+    max_queued INT NOT NULL,
+    soft_concurrency_limit INT,
+    hard_concurrency_limit INT NOT NULL,
+    scheduling_policy VARCHAR(128),
+    scheduling_weight INT,
+    jmx_export BOOLEAN,
+    soft_cpu_limit VARCHAR(128),
+    hard_cpu_limit VARCHAR(128),
+    parent BIGINT,
+    environment VARCHAR(128),
+    FOREIGN KEY (parent) REFERENCES resource_groups (resource_group_id) ON DELETE CASCADE
```

Comment:
Should there be indexes created for FK columns? It might be out of the scope of this PR though, as they are not present in the existing MySQL schema.

Reply:
I can do this in a follow up PR for MySQL/PostgreSQL/Oracle if that is OK with you.

Reply:
Resource group tables aren't expected to have enough entries to make the indexes useful.